components/ai_chat/resources/untrusted_conversation_frame/components/code_block/style.module.scss

```diff
@@ -22,6 +22,10 @@
   code {
     padding: var(--leo-spacing-s);
   }
+
+  font: var(--leo-font-large-regular);
+  font-size: 14px;
+  font-style: normal;
```

Comment:
Screenshots of the style update:

<img width="820" height="611" alt="Screenshot-2025-12-14_19-09" src="https://github.com/user-attachments/assets/f1eb770a-c194-4ac8-921c-e491266c0b40" />
<img width="837" height="1319" alt="Screenshot-2025-12-14_19-10" src="https://github.com/user-attachments/assets/c2f7e025-d313-4a69-ad50-6f0392cf5e64" />

Reply:
Awesome 👏 Thank you!